src/Microsoft.PowerShell.ConsoleHost/host/msh/ManagedEntrance.cs

```diff
@@ -33,16 +33,11 @@ public static int Start(string consoleFilePath, [MarshalAs(UnmanagedType.LPArray
         {
             System.Management.Automation.Runspaces.EarlyStartup.Init();
 
-            // Set ETW activity Id
-            Guid activityId = EtwActivity.GetActivityId();
-
-            if (activityId == Guid.Empty)
-            {
-                EtwActivity.SetActivityId(EtwActivity.CreateActivityId());
-            }
-
-            PSEtwLog.LogOperationalInformation(PSEventId.Perftrack_ConsoleStartupStart, PSOpcode.WinStart,
-                PSTask.PowershellConsoleStartup, PSKeyword.UseAlwaysOperational);
+#if !UNIX
+            // NOTE: On Unix, logging has to be deferred until after command-line parsing
+            // complete. On Windows, deferring the call is not needed.
+            PSEtwLog.LogConsoleStartup();
+#endif
```

Comment:
Same here. The config file overriding should be done through an environment variable until the command-line parsing is moved to an earlier point.

Reply:
Until we decide what changes we want, I think this is the simpler solution.